### PR TITLE
Release 2.4.0

### LINKED ISSUE #251: Time for a new release
Hello, maintainers! :wave:

By my calculations, it's time for a new release of jekyll-seo-tag. It's been over 2 months since the last release, v2.3.0.

What else is left to be done before a new release can be made? Please make sure to update History.markdown too if it's not already updated.

Thanks! :revolving_hearts: :sparkles:

### PR DESCRIPTION
**Pull request by DirtyF**
Fix #251

### CODE REVIEW

**Comment by DirtyF** [comment]
@benbalter  can't push on Rubygems, so I hand it over to you :shipit:

File: History.markdown [hunk 1]
```diff
@@ -1,13 +1,20 @@
 ## HEAD
 
+## 2.4.0 / 2017-12-04
+
+### Minor
+
+  * Add meta generator (#236)
+  * Strip null values from JSON-LD hash (#249)
+
```
Comment (benbalter) [inline History.markdown]: I believe we also closed all self-closing tags in the template.

Comment (pathawks) [reply History.markdown]: #246 Was merged without @Jekyllbot

Comment (DirtyF) [reply History.markdown]: 👍 Added to the History

**Pull request merged**